tests/E2E.Tests/PdsMesh/PdsMeshTests.cs

```diff
@@ -80,31 +86,42 @@ public async Task GivenMessageInInboxWithMultiplePatients_WhenPatientsAlreadyExi
 
         var messageContent = await File.ReadAllTextAsync("PdsMesh/Samples/MeshResponseMultiplePatients.csv");
 
-        var dateMeshMessageSent = DateTime.Now;
+        var dateMeshMessageSent = DateTime.UtcNow;
 
         await SendPdsMeshMessage(messageContent);
 
-        await AssertPdsMeshResponseMessageHandledCorrectly(messageContent, dateMeshMessageSent);
+        InvokePdsRetrieveMessages();
+
+        AssertPdsMeshResponseMessageHandledCorrectly(messageContent, dateMeshMessageSent);
+    }
+
+    private void InvokePdsRetrieveMessages()
+    {
+        var pdsRequest = Post("/internal/run/pds", null, null);
+        var runPdsRequest = ApiClient.Execute(pdsRequest);
```

Comment:
in a test? why?

Reply:
Couple of reasons, but primarily to prevent blocking the thread and to get a little bit of extra scalability as the number of tests increase. In particular, this is of interest when considering [running tests in parallel](https://xunit.net/docs/running-tests-in-parallel). 

And, to quote Stephen Cleary: "Why not? They're just as easy as synchronous methods. async Task unit tests methods have been supported by every major unit test framework since 2012." [ref](https://stackoverflow.com/a/33478404)